htcondor/sources/job_q_source_proxy.py

```diff
@@ -25,10 +25,17 @@ def acquire(self):
         Acquire jobs from the HTCondor Schedd
         :rtype: :obj:`~pd.DataFrame`
         """
+        logging.getLogger().debug("we are in source job_q_source_proxy acquire()")
+
         job_manifests = super(JobQSourceProxy, self).acquire()
-        if not set(PRODUCES).issubset(set(job_manifests.keys())):
-            raise RuntimeError('SourceProxy %s not configured with all dataproducts %s' % (type(self).__name__, PRODUCES))
-        return job_manifests
+
+        try:
+            if not set(PRODUCES).issubset(set(job_manifests.keys())):
+                raise RuntimeError(f"SourceProxy {type(self).__name__} not configured with all dataproducts {PRODUCES}")
+        except Exception:
```

Comment:
since this looks like a defensive `except`, can you add `# pragma: no cover` to the `except Exception:` line?